packages/devtools/pack-extension.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why ignore the extraneous dependnecies lint here ?

Reply:
'bestzip' should be devdependency not dependency. But I've tried:
- list 'bestzip' as devdependency
- exclude this file from tsconfig.json
None of the above worked :/ so I in the end just added an eslint disable